client/src/controllers/CloneController.ts

```diff
@@ -159,4 +159,25 @@ export class CloneController extends Controller<HTMLElement> {
     const content = template.content.firstElementChild?.cloneNode(true);
     return content instanceof HTMLElement ? content : null;
   }
+
+  /**
+   * If called with an event, finds the first error item in the minimap,
+   * scrolls to it, and focuses on it.
+   */
+
+  jumpToError(event?: Event) {
+    event?.preventDefault();
+    const firstErrorItem = document.querySelector(
+      '.w-minimap-item--error',
```

Comment:
When reworking this, I would recommend we use the selector `.error-message,.help-critical`, this would take the user to the FIELD that has the error, not the section on the minimap. Saving them a click.

I think this was the goal of the original issue.